buildtools/smoketests.proj

```diff
@@ -0,0 +1,90 @@
+<Project ToolsVersion="Current" DefaultTargets="smoke-tests" xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
+    <PropertyGroup>
+        <RootPath Condition="'$(RootPath)'==''">$(MSBuildProjectDirectory)</RootPath>
+
+        <!-- We need to skip recipes that use Docker when running on Windows CodeBuild -->
+        <ShouldRunDockerTests>true</ShouldRunDockerTests>
```

Comment:
nit: could you specify an inverse condition here? Its easy to mess this up if in the future someone mistakenly reorders them.

Reply:
Flipped in https://github.com/aws/aws-dotnet-deploy/pull/823/commits/c10ee39db77ccf1479dbff8a9583bf4536931195. 

(had to refresh myself on https://en.wikipedia.org/wiki/De_Morgan%27s_laws :-D )